tests/ha/fencing.pm

```diff
@@ -25,6 +25,14 @@ sub run {
     check_cluster_state;
     barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
 
+    # Give time for HANA to replicate the database
+    if (check_var('AUTOMATED_REGISTER', 'true')) {
+        # TODO: Fix
+        sleep 300;
```

Comment:
Please, scale timeout with `bmwqemu::scale_timeout()`. Could be helpful for when we use this test in slower HW.

Reply:
The proper fix here would be to detect if the database backup in the primary was replicated in the secondary instead of sleeping.

Reply:
So, scaling of the timeout is not needed here then?